### PR TITLE
API-37109-eliminate-dup-error-messages

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

- *Removes min and max lengths for telephone and adds to the description specifying the length.

## Related issue(s)

- [API-37200](https://jira.devops.va.gov/browse/API-37200)

## Testing done

- Postman (see screenshot)

## Screenshots
Screenshot 2024-06-06 at 1 32 02 PM

## What areas of the site does it impact?
	modified:   modules/claims_api/config/schemas/v2/526.json
	modified:   modules/claims_api/config/schemas/v2/generate_pdf_526.json

## Acceptance criteria

- []  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by mchristiansonVA — APPROVED** [review]
Tested locally via Postman and confirmed only one error message is displayed when veteranNumber telephone contains more than 10 digits. LGTM!

**Review by mchristiansonVA — APPROVED** [review]
Tested locally again to confirm single error message display; also confirmed documentation correctly updated. LGTM!

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 1]
```diff
@@ -1625,12 +1623,10 @@
                                   "additionalProperties": false,
                                   "properties": {
                                     "telephone": {
-                                      "description": "Primary phone of point of contact.",
+                                      "description": "Ten digit primary phone of point of contact.",
                                       "type": "string",
                                       "pattern": "^\\d{10}?$",
                                       "example": "5555555",
```
Comment (MadBomber) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:1629]: BTW the example does not show ten digits.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 2]
```diff
@@ -2978,12 +2972,10 @@
                                 "additionalProperties": false,
                                 "properties": {
                                   "telephone": {
-                                    "description": "Primary phone of point of contact.",
+                                    "description": "Ten digit primary phone of point of contact.",
                                     "type": "string",
                                     "pattern": "^\\d{10}?$",
                                     "example": "5555555",
```
Comment (MadBomber) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:2978]: btw: ditto, example not 10 digits.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 3]
```diff
@@ -4864,12 +4854,10 @@
                                   "additionalProperties": false,
                                   "properties": {
                                     "telephone": {
-                                      "description": "Primary phone of point of contact.",
+                                      "description": "Ten digit primary phone of point of contact.",
                                       "type": "string",
                                       "pattern": "^\\d{10}?$",
                                       "example": "5555555",
```
Comment (MadBomber) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:4860]: ditto

**Pull request merged**